README.md

```diff
@@ -8,3 +8,28 @@ Discussion of system concepts and high level processes are found in the
 [central wiki](https://github.com/privly/privly-organization/wiki).
 
 **For more information** on developing for this repository, read [development.md](https://github.com/privly/privly-jetpack/blob/master/development.md) in this directory.
+
```

Comment:
In a subsequent pull request, do you want to update the "About" section? Otherwise, I can take care of it. It can be modeled on [privly-chrome](https://github.com/privly/privly-chrome).